roles/disaster_recovery/callback_plugins/stdout.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime
```

Comment:
I think this should be moved to the plugins of the collection. 
Create `callback` folder in https://github.com/oVirt/ovirt-ansible-collection/tree/master/plugins and move it there.

Reply:
probably misunderstood my comment.
I was referring to the callback plugin, not the import.
I think this you should move the whole file to the `plugins/callback`.

Reply:
Done.
No misunderstanding, just didn't handle till now :)
Anyway, committed a separate commit named "moving "stdout.py" callbacks file".